apps/web/src/pages/Users/IndexUserPage/components/UserTableFilterDialog/UserTableFilterDialog.tsx

```diff
@@ -198,17 +197,6 @@ const UserTableFilterDialog = ({
                   options={optionsData.employmentDuration}
                 />
               </div>
-              <div data-h2-flex-item="base(1of1) p-tablet(1of2) laptop(1of3)">
-                <MultiSelectField
-                  id="jobLookingStatus"
-                  name="jobLookingStatus"
-                  label={formatMessage({
-                    defaultMessage: "Availability",
-                    id: "hOxIeP",
-                  })}
-                  options={optionsData.jobLookingStatus}
-                />
-              </div>
               <div data-h2-flex-item="base(1of1) p-tablet(1of2) laptop(1of3)">
```

Comment:
We removed the item but now there is an open space in the dialog which looks a little odd. Could we extend the duration and completion filters to be `1of2` to even things up? Or @Jerryescandon could you suggest a better layout for the filter?

<img width="770" alt="Screenshot 2023-06-19 at 10 09 31 AM" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/cf1483c1-18c0-4bd4-a4c0-50c9cfaa4b2d">

Reply:
That sounds good to me, just extend both to fill the space.

Reply:
Here they are, expanded to share the space.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/feb83724-df1d-48b6-9ccc-25ca27963aeb)
Ref: [tweak layout of filter](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6996/commits/0f83e40d7c17164bb83ee68cc941fd5a2a4e42da)